app/api/dao/user.py

```diff
@@ -219,7 +219,7 @@ def update_user_profile(user_id: int, data: Dict[str, str]):
 
             # username should be unique
             if user_with_same_username:
-                return messages.USER_USES_A_USERNAME_THAT_ALREADY_EXISTS, HTTPStatus.BAD_REQUEST
+                return messages.USER_USES_A_USERNAME_THAT_ALREADY_EXISTS, HTTPStatus.CONFLICT
```

Comment:
undo this change.

Reply:
but it is according to maya saud,
409 has to be written when user email already exists